apps/aehttp/src/aehttp_app.erl

```diff
@@ -121,14 +121,19 @@ get_and_parse_ip_address_from_config_or_env(CfgKey, App, EnvKey, Default) ->
     {ok, IpAddress} = inet:parse_address(binary_to_list(Config)),
     IpAddress.
 
-get_http_api_acceptors(_) -> ?INT_ACCEPTORS_POOLSIZE.
+get_http_api_acceptors(external) ->
+    aeu_env:user_config_or_env([<<"http">>, <<"external">>, <<"acceptors">>],
```

Comment:
I suggest enabling user to set this as per WS https://github.com/aeternity/epoch/blob/4795b2c820e1f44c23a92cb62ef5c85ec47cbd9e/apps/aeutils/priv/epoch_config_schema.json#L151

see also https://github.com/aeternity/epoch/blob/4795b2c820e1f44c23a92cb62ef5c85ec47cbd9e/apps/aeutils/test/data/epoch_full.yaml#L39

Reply:
✅